lib/utils.py

```diff
@@ -57,14 +57,24 @@ def validate_url(string):
 
 def get_node_ip():
     if arch in ('armv6l', 'armv7l'):
-        active_interfaces = interfaces()
 
-        if 'eth0' in active_interfaces:
-            interface = 'eth0'
-        elif 'wlan0' in active_interfaces:
-            interface = 'wlan0'
-        else:
-            raise Exception("No active network interface found.")
+        interface = None
+        for ifc in ['eth0', 'wlan0']:
```

Comment:
I don't think this works on `wlan0`? It's a physical link, so I don't think this concept is applicable.